packages/redirections/graphql/schema.js

```diff
@@ -1,9 +1,35 @@
 module.exports = `
 schema {
   query: queries
+  mutation: mutations
 }
 
 type queries {
   redirection(path: String): Redirection
+  redirections(
+    limit: Int,
+    offset: Int
+  ): [Redirection!]!
```

Comment:
Above pattern is used in queries like `adminUsers`, `payments` or `postfinancePayments`. It is more efficient than loading all nodes (retrieval) and slicing them (memory).

Not eager to change it but since I found `@oribiting/backend-modules-utils/paginate` that should be fairly easy to do.

Do you think I should switch it?

Reply:
[In this commit on a separate branch](https://github.com/orbiting/backends/commit/f040eb7bc8d9e505a79f2af5a4c2abcd71f7e281)we're using pagination pattern for redirections now.

Not convinced it is a good idea to load all nodes and then slice them.

Reply:
Granted, this implementation is not perfect. But until now it happend many times, that not paginating an endpoint resultet in a slow frontend over time and the implementation of pagianation in the backend. But slicing in memory was never a problem until now :smirk:  So for most cases loading eeeverything to memory and slicing it there is simple and fast enough. Until it's not anymore, but then we have one central point to improve the implementation for all endpoints. The commit looks good and simple I would go for it.
The only argument against it that I can see now is that this API endpoint is inteded to be used via graphiql (without a frontend) for now and the handling pagination by hand is cumbersome. And pagination can be added easily later as your commit shows.
So I'm fine with both ways.